src/rfc3161_client/__init__.py

```diff
@@ -31,4 +31,4 @@
     "SignerInfo",
 ]
 
-__version__ = "0.0.1"
```

Comment:
FWIW I'm OK with just removing this -- if we can't single-source the version, the `__version__` attribute is less important (since it's just a convention).